app.py

```diff
@@ -53,6 +53,11 @@ def hello_world():
 
 @app.route('/resume/experience', methods=['GET', 'POST'])
 def experience():
+
+    if request.method == 'GET':
+        return jsonify([exp.serialize() for exp in data['experience']])
+    return jsonify({})
+
```

Comment:
your change is breaking `GET` by id

Please look further down the file to find this part of code:

```python
if request.method == 'GET':
        return jsonify([
            {k: v for k, v in exp.__dict__.items() if k != "id"}
            for exp in data["experience"]
        ])
```

please make sure that it doesn't break the existing code